### DISCUSSION TITLE
Computer Networking: A Top-Down Approach - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#112] [Announcements]
# Computer Networking: A Top-Down Approach - CS自学指南

CS自学指南

https://csdiy.wiki/%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%BD%91%E7%BB%9C/topdown/

**Comment by Laporte12138** [comment]
北邮同学前来插眼

**Comment by tategotoazarasi** [comment]
这个也不错：
中科大郑烇、杨坚全套《计算机网络（自顶向下方法 第7版，James F.Kurose，Keith W.Ross）》课程
https://www.bilibili.com/video/BV1JV411t7ow/

**Reply by zbbBella** [reply]
+10086 我也想说这个！！
我愿称之为计网最好的中文课程

**Reply by qiuqiumaomao** [reply]
顺便学学英语，对计算机学生来说很重要吧：）

**Reply by yhlincs** [reply]
你好，请问有看过哈工大那个计网课程吗？初学的话这两个看哪一个好？？

**Reply by EnzoGuang** [reply]
还是中科大郑烇老师的好

**Comment by qiuqiumaomao** [comment]
这门课这么感觉课时比国内少很多，1/3有了吧

**Reply by qiuqiumaomao** [reply]
讲的比较浅，而且章节练习题难度提升了，主要是很多课上都没讲

**Reply by tategotoazarasi** [reply]
国内的40小时，这个10小时

**Comment by Laporte12138** [comment]
我觉得中科院的这个

在 2022-06-24 10:54:55，yhlin ***@***.***> 写道：


你好，请问有看过哈工大那个计网课程吗？初学的话这两个看哪一个好？？

**Reply by yhlincs** [reply]
谢谢哈，我正在看哈工大的，感觉也挺不错的，那个是中科大不是中科院哦。。。。。。。。

**Comment by Laporte12138** [comment]
笔误了......okok
















在 2022-07-07 15:45:24，"yhlin" ***@***.***> 写道：

谢谢哈，我正在看哈工大的，感觉也挺不错的，那个是中科大不是中科院哦。。。。。。。。
